src/styles/blocks.scss

```
@@ -0,0 +1,26 @@
+.blocks {
+  display: grid;
+  grid-template-columns: 1fr;
+
+  @media (min-width: 2 * $block-width) and (max-width: 3 * $block-width - 1px) {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  @media (min-width: 3 * $block-width) {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
```

You rewrite same rule, it should work without `max-width` setting
```suggestion
  @media (min-width: 2 * $block-width) {
    grid-template-columns: 1fr 1fr;
  }

  @media (min-width: 3 * $block-width) {
    grid-template-columns: 1fr 1fr 1fr;
  }
```